data/sheragi/the emerald sword.txt

```diff
@@ -0,0 +1,854 @@
+
+mission "Sheragi Archaeology: The Emerald Sword 1"
+	landing
+	name "Black Box Decoded"
+	description "Visit Albert Foster on Zug."
+	source
+		near "Zubeneschamali" 1 100
+	destination "Zug"
+	
+	to offer
+		has "event: cube decoded"
+	
+	on offer
+		dialog
+			`You're still in the middle of the post-flight check when you receive a new message from Albert Foster.`
+			`	"Hello <first>. I have some good news! We have decoded what's in the black box. You should come visit us on Zug; we have plenty to tell you!"`
+
+mission "Sheragi Archaeology: The Emerald Sword 2"
+	landing
+	name "The Emerald Sword"
+	description "Bring an additional Jump Drive to Zug."
+	source "Zug"
+	to offer
+		has "Sheragi Archaeology: The Emerald Sword 1: done"
+	
+	on offer
+		conversation
+			`You arrive on Zug, and after contacting Albert and the three Hai archaeologists and following their direction, you find them in an ice cream shop. They're still wearing the youngster clothes, but not even bothering to hide their faces with bandannas anymore. Surprisingly, nobody seems too distressed at the sight. Rakei waves for you and stands up, coming to you with ice creams in paw.`
+			`	"Hello <first>! It seems our disguise worked better than intended. The humans around here think we're wearing a costume. Apparently young humans like to dress up as aliens?" she says confusedly.`
+			`	"I had a feeling we'd fit right in with the furry crowd." Eyama responds proudly.`
+			`	"It has been an interesting cultural exchange. Human space might be a great vacation spot for us. But we're not on vacation yet. We have a mystery to solve," she continues.`
+			`	You sit down with your group on a bench in a nearby park, mostly empty, and Desyo picks up the conversation.`
+			`	"So, the files in the black box were thankfully standard for Sheragi. Not very difficult to decode, once we managed to read them."`
+			choice
+				`	"What did it contain?"`
+			`	Desyo responds, "It was all fleet movement data and military strategy details. This system was apparently on the border of one of the faction's territories. On the final moments of the recording, a large transport fleet had entered the system, unarmed ships. They were followed by military vessels in pursuit. There's no telling what they thought at the time, but a military fleet was immediately dispatched to eliminate both fleets. One particular ship, which Eyama has translated the name as 'Emerald Sword,' started behaving erratically when given orders. It fired and missed a few shots until it eventually stopped and fled. Clear signs of a mutiny. However, what is perhaps most interesting is that when it fled, it fled in an unexpected direction, away from any hyperlanes."`
```

Comment:
**ERROR**
`which Eyama has translated the name as 'Emerald Sword,'`
This should change to `whose name Eyama has translated to 'Emerald Sword',`